text/2996-async-iterator.md

```diff
@@ -1,69 +1,69 @@
-- Feature Name: `async_stream`
+- Feature Name: `async_iterator`
 - Start Date: 2020-09-29
 - RFC PR: [rust-lang/rfcs#2996](https://github.com/rust-lang/rfcs/pull/2996)
 - Rust Issue: [rust-lang/rust#79024](https://github.com/rust-lang/rust/issues/79024)
 
 # Summary
 [summary]: #summary
 
-Introduce the `Stream` trait into the standard library, using the
-design from `futures`. Redirect the `Stream` trait definition in the 
+Introduce the `AsyncIterator` trait into the standard library, using the
+design from `futures`. Redirect the `AsyncIterator` trait definition in the 
 `futures-core` crate (which is "pub-used" by the `futures` crate) to the standard library.
```

Comment:
The `futures-core` crate still calls it `Stream` today, so maybe this should read something like:

```suggestion
design from `futures`. Redirect the `Stream` trait definition in the 
`futures-core` crate (which is "pub-used" by the `futures` crate) to the `AsyncIterator` trait in the standard library.
```

Reply:
Updated; thanks!